src/sxg_cbor.c

```diff
@@ -50,7 +50,7 @@ bool sxg_write_utf8_cbor_header(uint64_t length, sxg_buffer_t* target) {
   return write_initial_bytes(0x60, length, target);
 }
 
-bool sxg_write_map_cbor_header(uint64_t size, sxg_buffer_t* target) {
+bool sxg_write_map_cbor_header(size_t size, sxg_buffer_t* target) {
```

Comment:
`write_initial_bytes` takes a `uint64_t` length.  Would it be better to change the header declaration for `sxg_write_map_cbor_header` to `uint64_t` instead?

Reply:
Yeah, that probably makes more sense. Updated.

Reply:
Looks like all the callers of `sxg_write_map_cbor_header` pass in a `size_t`. For defense-in-depth, it would be nice to add an `if (value < 0) do_something_else();` before each one is cast to a `uint64_t`, but I'll file a bug for that. Wasn't introduced by this PR.